src/Filters/molecules/FilterDropdown.tsx

```diff
@@ -34,22 +34,28 @@ const StyledFilterOption = styled(FilterOption)`
   letter-spacing: 0.2px;
 `;
 
-const OptionList = styled.div`
+const OptionList = styled.div<{isChromeOrSafari: boolean}>`
   max-height: 162px;
   min-height: 40px;
   position: relative;
   overflow-y: auto;
-  margin-right: 5px;
+  
+  /* Firefox supports the scrollbar-width property */
+  scrollbar-width: thin;
 
-  /* Customize the scrollbar */
+  /* Apply margin-right to scrollbar for Chrome and Safari */
+  ${({ isChromeOrSafari }) => isChromeOrSafari && `
```

Comment:
It is ok to have 2px margin for all browser to avoid this condition.

Reply:
For Firefox, it is by default taking some right margin
to maintain uniformity i have added right margin to chrome and safari
Please check below attached screenshot If I remove right margin 
![image](https://github.com/future-standard/scorer-ui-kit/assets/118800413/c4826932-5eb7-4b6f-8437-cb46da23507e)
![image](https://github.com/future-standard/scorer-ui-kit/assets/118800413/73a788c0-2777-42e5-a1ef-c469da5ee260)

If I'll add right margin without any browser specific condition, then extra right margin will be added to the firefox, will it be okay?????
please check below attached screenshot

![image](https://github.com/future-standard/scorer-ui-kit/assets/118800413/71bc1284-af25-418a-9e9b-ede3fc3eae61)
![image](https://github.com/future-standard/scorer-ui-kit/assets/118800413/94833af0-cc31-44e7-b291-9acc8e288624)

Reply:
Done